Apps/WebClient/src/ClientApp/src/components/report/medicationHistory.vue

```diff
@@ -200,17 +189,17 @@ export default class MedicationHistoryReportComponent extends Vue {
 
 <template>
     <div>
-        <div ref="report">
+        <div>
             <section class="pdf-item">
                 <ReportHeaderComponent
-                    v-show="!isPreview"
+                    v-show="!isLoading"
                     :filter="filter"
                     :title="
                         'Health Gateway Medication History' +
                         (filter.hasMedicationsFilter() ? ' (Redacted)' : '')
                     "
                 />
```

Comment:
The header didn't used to be shown during the preview.  We're always previewing now, so the header can be removed completely.

Reply:
removed